modules/kafka/src/main/java/org/testcontainers/containers/KafkaContainer.java

```diff
@@ -136,6 +144,9 @@ protected void configure() {
     }
 
     protected void configureKraft() {
+        //CP 7.4.0
+        withEnv("KAFKA_CONTROLLER_QUORUM_MODE", "kraft");
+        withEnv("CLUSTER_ID", getEnvMap().computeIfAbsent("CLUSTER_ID", key -> clusterId));
```

Comment:
I think we can use it as 

```suggestion
        getEnvMap().computeIfAbsent("CLUSTER_ID", key -> clusterId);
```

Reply:
yes, please